Microsoft.Toolkit.Uwp.UI.Controls/LayoutTransformer/LayoutTransformer.cs

```diff
@@ -0,0 +1,537 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
```

Comment:
@hawkerm Good question. I checked and the code in blog post is originally from Silverlight XAML, so it is already the property of Microsoft. So, I suppose we can.